app/templates/documents.rb

```diff
@@ -167,7 +167,7 @@ def filter_array(locale)
 
   def properties(json)
     json.properties do
-      %w(updated created).each { |field| date(json, field) }
+      %w[updated created changed].each { |field| date(json, field) }
```

Comment:
The lack of the `changed` field in our set of timestamp fields was an oversight. The templates will be updated further in the next story.